### PR TITLE
Feature/custom 404 page

### PR DESCRIPTION
**Pull request by edumoreira1506**
Before of this, the 404 page was static, and link to back to navigation didn't work. Preview:
Screenshot from 2020-01-10 12-10-49

### CODE REVIEW

File: app/views/errors/not_found.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <title>404 - <%= t('not_found.title') %> - <%= t('cm-central') %></title>
+  <%= csrf_meta_tag %>
+  <%= favicon_link_tag "favicon.ico" %>
+  <%= stylesheet_link_tag :application %>
+</head>
+<body class="NotFound">
+  <div class="NotFound__container">
+    <h2 class="NotFound__title">404</h2>
+    <h3 class="NotFound__description"><%= t('not_found.description') %></h3>
+    <%= link_to t('back'), root_path, class: "NotFound__link" %>
```
Comment (talyssonoc) [inline app/views/errors/not_found.html.erb]: I think we're missing the back reference here (and then fallback to `root_path`):

```suggestion
    <%= link_to t('back'), request.referer || root_path, class: "NotFound__link" %>
```

Comment (edumoreira1506) [reply app/views/errors/not_found.html.erb]: Oh thanks!

**Comment by talyssonoc** [comment]
I wasn't able to access the 404 page in other languages, even if I switch to Portuguese it shows the error message in English. Can you check?

File: app/views/errors/not_found.html.erb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>404 - <%= t('not_found.title') %> - <%= t('cm-central') %></title>
+  <%= csrf_meta_tag %>
```
Comment (talyssonoc) [inline app/views/errors/not_found.html.erb]: Do we need CSRF in the error page?

Comment (edumoreira1506) [reply app/views/errors/not_found.html.erb]: We don't need, I will remove.

**Pull request merged**